crates/task-impls/src/quorum_vote/drb_computations.rs

```diff
@@ -0,0 +1,105 @@
+use std::collections::BTreeMap;
+
+use hotshot_types::{
+    drb::{compute_drb_result, DrbResult, DrbSeedInput},
+    traits::node_implementation::NodeType,
+};
+use tokio::{spawn, task::JoinHandle};
+
+/// Helper struct to track state of DRB computations
+pub struct DrbComputations<TYPES: NodeType> {
+    /// Stored results from computations
+    results: BTreeMap<TYPES::Epoch, DrbResult>,
+
+    /// Currently live computation
+    task: Option<(TYPES::Epoch, JoinHandle<DrbResult>)>,
+
+    /// Stored inputs to computations
+    seeds: BTreeMap<TYPES::Epoch, DrbSeedInput>,
+}
+
+impl<TYPES: NodeType> DrbComputations<TYPES> {
```

Comment:
Looks like we also need a function to insert a seed, which will happen at the beginning of each epoch. Then we can use the seed in the next epoch to start the computation.